website/scripts/build-static-content.js

```diff
@@ -517,6 +517,135 @@ module.exports = {
+                //   // If any columns are missing a type or description, we'll throw an error.
+                //   throw new Error(`Could not merge osquery schema with Fleet schema JSON, a column is missing a description. to resolve, add a column description to the column in the fleet_schema.JSON file. Column that caused this error: ${column.name} in the ${table.name} table`);
+                // }
+                // Add the table row for this column to the markdown string.
+                tableMdString += ' | '+column.name+' | '+ column.type +' | '+columnDescriptionForTable+'|\n';
+              }
+            }
+            if(table.examples) { // If this table has a examples value (These will be in the Fleet schema JSON) We'll add the examples to the markdown string.
+              // table.examples = table.examples.replace(/(```)([a-zA-Z0-9\-]*)(\s*\n)/g, '$1\n' + '<!-- __LANG=%' + '$2' + '%__ -->' + '$3'); // « Based on the github-flavored markdown's language annotation, (e.g. ```js```) add a temporary marker to code blocks that can be parsed post-md-compilation when this is HTML.  Note: This is an HTML comment because it is easy to over-match and "accidentally" add it underneath each code block as well (being an HTML comment ensures it doesn't show up or break anything).  For more information, see https://github.com/uncletammy/doc-templater/blob/2969726b598b39aa78648c5379e4d9503b65685e/lib/compile-markdown-tree-from-remote-git-repo.js#L198-L202
+
+              tableMdString += '\n### Example\n\n'+table.examples+'\n';
+            }
+            if(table.notes) { // If this table has a examples value (These will be in the Fleet schema JSON) We'll add the examples to the markdown string.
+              tableMdString += '\n### Notes\n\n'+table.notes+'\n';
+            }
+            // Determine the htmlId for table
+            let htmlId = (
+              _.last(table.name.split(/\//)).slice(0,20)+
+              '--'+
+              sails.helpers.strings.random.with({len:10})
+            ).replace(/[^a-z0-9\-]/ig,'');
+
+            // Convert the markdown string to HTML.
+            let htmlString = await sails.helpers.strings.toHtml(tableMdString);
+
+            // Add the language-sql class to codeblocks in generated HTML partial for syntax highlighting.
+            htmlString = htmlString.replace(/(<pre><code)([^>]*)(>)/gm, '$1 class="language-sql"$2$3');
+            // Determine the path of the folder where the built HTML partials will live.
+            let htmlOutputPath = path.resolve(sails.config.appPath, path.join(APP_PATH_TO_COMPILED_PAGE_PARTIALS, htmlId+'.ejs'));
+            if (dry) {
+              sails.log('Dry run: Would have generated file:', htmlOutputPath);
+            } else {
+              await sails.helpers.fs.write(htmlOutputPath, htmlString);
+            }
+            // Add this table to the array of schemaTables in builtStaticContent.
+            builtStaticContent.schemaTables.push({
```

Comment:
I used a new array because of existing behavior on `basic-documentation.page.js` that expects objects in the `builtStaticContent.markdownPages` array to have a `sectionRelativeRepoPath` value. 

I will update the script to add the tables to the `markdownPages` array and adjust it so the schema is processed after the Markdown content has been built.